src/server/system_services/system_store.js

```diff
@@ -410,8 +410,12 @@ class SystemStore extends EventEmitter {
                     dbg.log0('SystemStore.load: Got load request with a timestamp older then my last update time');
                     this.last_update_time = since;
                 }
-
-                this.master_key_manager.load_root_key();
+                const { NOOBAA_ROOT_SECRET } = process.env;
+                if (NOOBAA_ROOT_SECRET) {
+                    this.master_key_manager.load_root_key();
+                } else {
+                    await this.master_key_manager.load_root_keys_from_mount();
```

Comment:
As discussed in an online review:
* we can take the `load_root_keys_from_mount` outside to `initial_load`
   * on system_store load this should not matter since the system key was already decrypted and is inside the cache in `master_key_manager`
   * This is not a complete solution for when we will want to rotate the system key. in that case we will also want to do `load_root_keys_from_mount`
* in the bg worker, we will call `load_root_keys_from_mount` in every iteration